src/components/backend-ai-environment-list.ts

```diff
@@ -965,9 +965,8 @@ export default class BackendAIEnvironmentList extends BackendAIPage {
         );
     } else {
       this.modifyImageCudaGpu.label = _t('environment.Disabled') as string;
-      (
-        this.shadowRoot?.querySelector('mwc-slider#cuda-gpu') as Slider
-      ).value = 0;
+      (this.shadowRoot?.querySelector('mwc-slider#cuda-gpu') as Slider).value =
+        0;
```

Comment:
This PR includes several changes to update the code format only. I believe this was caused by https://github.com/lablup/backend.ai-webui/pull/2162. After this, the formatting rules have slightly changed. In that PR, we should have reapplied the rules to all files. I will create a separate issue to reapply r format rule to all files to accommodate the slightly altered rules.